packages/vaadin-app-layout/test/app-layout.test.js

```diff
@@ -234,4 +237,81 @@ describe('vaadin-app-layout', () => {
       });
     });
   });
+
+  describe('notify children about resize', () => {
+    class ResizeAwareElement extends mixinBehaviors([IronResizableBehavior], PolymerElement) {
+      static get is() {
+        return 'resize-aware';
+      }
+    }
+
+    customElements.define(ResizeAwareElement.is, ResizeAwareElement);
+
+    let resizeAwareChild;
+
+    beforeEach(async () => {
+      layout = fixtureSync(`
+        <vaadin-app-layout>
+          <vaadin-drawer-toggle slot="navbar"></vaadin-drawer-toggle>
+          <h2 slot="navbar">App name</h2>
+          <section slot="drawer">
+            <p>Item 1</p>
+            <p>Item 2</p>
+            <p>Item 3</p>
+          </section>
+          <resize-aware></resize-aware>
+        </vaadin-app-layout>
+      `);
+      // force non-overlay mode as default
+      layout.style.setProperty('--vaadin-app-layout-drawer-overlay', 'false');
+      layout._updateOverlayMode();
+      await nextFrame();
+      resizeAwareChild = layout.querySelector('resize-aware');
+      sinon.stub(resizeAwareChild, 'notifyResize');
+    });
+    afterEach(() => {
```

Comment:
nit: please add a newline before this line.

Reply:
Added.